### ISSUE TITLE
[UI-Shell] axe landmark-contentinfo-is-top-level failure

### ISSUE THREAD

**Issue by yoshiemuranaka** [Qiskit/web-components#155]
### Steps to reproduce the problem

In platypus ui-shell integration [PR](https://github.com/Qiskit/platypus/actions/runs/3062612167/jobs/4943782319), [remove disabled rule](https://github.com/Qiskit/platypus/pull/1555/files#diff-6318d86254fdf33e8832e006c9b06f494bec88a0101f7a34b41b17558b056d73R42) and run `npm run test` and see failures.

[Rule link](https://dequeuniversity.com/rules/axe/4.4/landmark-contentinfo-is-top-level?application=axeAPI)

Changing the sidenav social icon [`<footer>`](https://github.com/Qiskit/web-components/blob/main/components/ui-shell/index.ts#L67) to a regular `<div>` will resolve the problem

```node
1 accessibility violation was detected
{
  id: 'landmark-contentinfo-is-top-level',
  impact: 'moderate',
  tags: [ 'cat.semantics', 'best-practice' ],
  description: 'Ensures the contentinfo landmark is at top level',
  help: 'Contentinfo landmark should not be contained in another landmark',
  helpUrl: 'https://dequeuniversity.com/rules/axe/4.4/landmark-contentinfo-is-top-level?application=axeAPI',
  nodes: [
    {
      any: [Array],
      all: [],
      none: [],
      impact: 'moderate',
      html: '<footer class="qiskit-side-nav-footer">',
      target: [Array],
      failureSummary: 'Fix any of the following:\n' +
        '  The null landmark is contained in another landmark.'
    }
  ]
}
┌─────────┬─────────────────────────────────────┬────────────┬────────────────────────────────────────────────────┬───────┐
│ (index) │                 id                  │   impact   │                    description                     │ nodes │
├─────────┼─────────────────────────────────────┼────────────┼────────────────────────────────────────────────────┼───────┤
│    0    │ 'landmark-contentinfo-is-top-level' │ 'moderate' │ 'Ensures the contentinfo landmark is at top level' │   1   │
└─────────┴─────────────────────────────────────┴────────────┴────────────────────────────────────────────────────┴───────┘
    1) has no detectable violations on load


  0 passing (9s)
  1 failing

  1) Accessibility
       has no detectable violations on load:
     AssertionError: 1 accessibility violation was detected: expected 1 to equal 0
      at Context.eval (http://localhost:8080/__cypress/tests?p=cypress/support/index.js:797:20)
```

### What is the current behavior?

Running platypus cypress tests fails accessibility spec with new ui-header component

### What is the expected behavior?

The existing a11y tests should pass

### Screenshots

![Accessibility -- has no detectable violations on load (failed)](https://user-images.githubusercontent.com/8097306/190507363-18f7278e-d274-4454-95bc-753365a42310.png)

### Additional information

Changing the sidenav social icon [`<footer>`](https://github.com/Qiskit/web-components/blob/main/components/ui-shell/index.ts#L67) to a regular `<div>` will resolve the problem
